ci/Jenkinsfile-kubernetes

```diff
@@ -344,6 +401,7 @@ node("docker") {
     def props = readPropertiesFile("gradle.properties")
     def sparklingVersion = props['version'].replace("-SNAPSHOT", "")
     
+/*
```

Comment:
Do we need this stage if it's commented out?

Reply:
It is just for testing so I don't have to wait for new cluster and images to be build-> I will uncoment before merging

Reply:
Removed